projects/prison-case-notes-to-probation/src/main/resources/application.yml

```diff
@@ -39,6 +39,8 @@ management:
 logging.level:
   org.hibernate.engine.jdbc.spi.SqlExceptionHelper: OFF
 
+ignore.message: NOMIS Alert Security. Do not share with offender and OCG Nominal - Do not share made active.
```

Comment:
We're probably fine to hard-code this in the Kotlin code for simplicity - it seems like it will be relatively static in each environment.

Reply:
will update

Reply:
done